pkg/auth/handler/userverify/verify_request.go

```diff
@@ -280,7 +278,33 @@ func (h VerifyRequestTestHandler) DecodeRequest(request *http.Request) (handler.
 }
 
 func (h VerifyRequestTestHandler) Handle(req interface{}) (resp interface{}, err error) {
-	// TODO: implement test sending
+	payload := req.(VerifyRequestTestPayload)
+	sender := h.TestCodeSenderFactory.NewTestCodeSender(
+		payload.Provider,
+		payload.ProviderConfig,
```

Comment:
I think the test handler should reuse the configured provider instead of accepting configuration of provider to build a new one on the fly. The main use case of the test endpoint is to send an email or SMS to see how it looks.